ooiui/static/js/partials/StreamDownloadForm.html

```diff
@@ -84,7 +87,7 @@
         <div class="col-md-6">
           <div class="form-group" id="type-select">
             <label for="type-select">File Type</label>
-            <select class="form-control">
+            <select class="form-control" title="Currently downloads are only available from the data source as NetCDF format">
```

Comment:
Grammar Freak.  Sorry.

"Currently, downloads are only available from the data source in NetCDF format"